blockstack-sdk/src/main/java/org/blockstack/android/sdk/BlockstackSession.kt

```diff
@@ -185,7 +182,7 @@ class BlockstackSession(context: Context? = null, private val config: Blockstack
         v8.add("console", v8Console)
         v8Console.registerJavaMethod(console, "log", "log", arrayOf<Class<*>>(String::class.java))
         v8Console.registerJavaMethod(console, "error", "error", arrayOf<Class<*>>(String::class.java))
-        v8Console.registerJavaMethod(console, "error", "error", arrayOf<Class<*>>(V8Object::class.java))
+        v8Console.registerJavaMethod(console, "error", "error", arrayOf<Class<*>>(String::class.java))
```

Comment:
That is the same as above. Sometimes `console.log(error)` is called with an error object.